### DISCUSSION TITLE
Thank you for Visidata

### DISCUSSION THREAD

**Discussion by halloleo** [saulpw/visidata#2659] [❤️]
This is my 3rd or 4th attempt to get into `vd` -- and I'm still struggling with both the logic of the program and the many key combos. But for the first time I feel: 

**Editing and cleaning CSVs with VisiData _works_ (including finding in-app help)!**

So thanks heaps for this cool software!

**Comment by saulpw** [comment]
You're welcome, @halloleo! I'm glad you're getting into it.  Can you share some of the struggles you're having with the logic, as you're experiencing them?  We can try to help you figure it out, and also it might help us make some improvements in the future.

**Comment by halloleo** [comment]
Thanks @saulpw.

Will reply with more details in due course.

Happy New Year for now!

**Reply by cool-RR** [reply]
I recommend this tutorial: https://jsvine.github.io/intro-to-visidata/

Happy new year!

**Comment by midichef** [comment]
Thanks for taking time to give feedback!

I too would like to hear the specifics of your visidata learning. Whatever would have helped you, would be good to put in guides and demos to help other people.

What features or keys did you find that made you think "wow, that's so useful, I wish I'd discovered that sooner"?
Or "I wish that had been explained differently?"
